assets/src/scripts/components/hydrograph/index.js

```diff
@@ -289,12 +289,20 @@ const controlLastYearSelect = function(elem, {compareTimeseries}) {
     }
 };
 
+const createTitle = function(elem) {
```

Comment:
I'm wondering if it would be more consistent with the rest of the module if this function took the `titleSelector` as an argument instead. But whatever, it's six of one and half dozen of the other.

Reply:
Several use this same pattern (e.g., createTooltipText). I want the location of this div to only be rendered once, so there are no surprises as to where it is in the markup. Doing this way means only the contents of the div change.